templates/zerver/help/invite-new-users.md

```diff
@@ -1,162 +1,90 @@
 
-1. Click **Save changes**.
+    **Note**: The number of email invites you can send in a day is
+    limited on the Zulip Cloud Free plan. [Contact support](/help/contact-support)
+    if you hit the limit and want to invite more users.
 
 {end_tabs}
 
-Before anyone joins your organization this way, we'll send a validation link
-to verify their email address.
-
-## Send invitations
+## Send an invitation link
 
-By default, organization admins and members can send
-invitations. Organization admins can also change who can send invitations.
-
-Note that on most Zulip servers (including Zulip Cloud), email invitations
-and reusable invitation links expire 10 days after they are sent.
+{!admin-only.md!}
 
 {start_tabs}
 
-{tab|send-email-invitations}
-
-{relative|gear|invite}
-
-1. Enter a list of email addresses.
-
-1. Decide whether the users should join as [owners, admins, moderators,
-   members, or guests](/help/roles-and-permissions).
-
-1. Select which streams they should join. If you send invitations often, you
-   may want to configure a set of
-   [default streams](/help/set-default-streams-for-new-users).
-
-1. Click **Invite**.
-
-!!! warn ""
-
-    * You will only see **Invite users** in the gear menu if you have
-    permission to invite users.
-    * The number of email invites you can send in a day is limited in
-    the free plan. [Contact us](/help/contact-support) if you hit the
-    limit and want to invite more users.
-
-{tab|share-an-invite-link}
-
 {relative|gear|invite}
 
 1. Click **Generate invite link**.
 
-1. Decide whether users using the link should join as [owners, admins, moderators
-   members, or guests](/help/roles-and-permissions).
+1. Select when the invitation will expire.
+
+1. Select what [role](/help/roles-and-permissions) the users will join as.
 
-1. Select which streams they should join. If you send invitations often, you
-   may want to configure a set of
-   [default streams](/help/set-default-streams-for-new-users).
+1. Configure which streams they will be added to.
 
 1. Click **Generate invite link**.
 
 1. Copy the link, and send it to anyone you'd like to invite.
 
 !!! warn ""
-
-    * You will only see **Invite users** in the gear menu if you have
-    permission to invite users.
-    * Only organization administrators can create these reusable invitation links.
-
-
-{end_tabs}
-
-## Change who can send invitations
-
-{!owner-only.md!}
-
-By default, all members can invite new users to join your Zulip organization.
-However, you can restrict the ability to do so to specific
-[roles](/help/roles-and-permissions).
-
-{start_tabs}
-
-{settings_tab|organization-permissions}
-
-1. Under **Joining the organization**, configure
-   **Who can invite users to this organization**.
-
-1. Click **Save changes**.
+    **Warning**: Creating an account using an invitation link does not require the user
+    to authenticate using a [configured authentication
+    method](/help/configure-authentication-methods).
```

Comment:
I'll double-check, but I think this is only true for the single-use invitations sent to a given email address.

Reply:
OK, cool, let's make sure this is accurate and is referenced in the right instruction blob.

Reply:
Confirmed, this is what you get when clicking reusable invitation links. I'll move this warning to be only for the block where it applies.

![image](https://user-images.githubusercontent.com/2746074/160501479-5691271e-6c06-41f1-bae9-3f704c9943b4.png)

Reply:
I think medium-term, we want to change the non-reusable invitation links to take you to this experience if and only if the default EmailAuthBackend is not enabled.